internal/extsvc/bitbucketcloud/client.go

```diff
@@ -234,8 +233,8 @@ func (c *client) do(ctx context.Context, req *http.Request, result any) error {
 		nethttp.ClientTrace(false))
 	defer ht.Finish()
 
-	if err = c.rateLimit.Wait(ctx); err != nil {
-		return err
+	if err := c.rateLimit.Wait(ctx); err != nil {
```

Comment:
I think the added `:` is unnecessary here:

```suggestion
	if err = c.rateLimit.Wait(ctx); err != nil {
```